doc/api-docs/installing.rst

```diff
@@ -5,122 +5,74 @@ Installing the InVEST Python Package
+2. As a binary wheel or source package from the `Python Package Index <https://pypi.org/project/natcap.invest/>`_
+3. As a source installation, directly from our `git source tree <https://github.com/natcap/invest/>`_
 
+Note that installing the python package does not include a user interface.
 
-**Condensed into one line**::
+Suggested Method: ``conda-forge``
+---------------------------------
 
-    conda create -y -c conda-forge -n invest391 python=3.9.7 gdal=3.3.1 && conda activate invest391 && pip install natcap.invest==3.9.1
+The easiest way to install ``natcap.invest`` is using the
+`conda <https://docs.conda.io/en/latest/miniconda.html>`_ or
+`mamba <https://mamba.readthedocs.io/en/latest/installation.html>`_ package managers::
 
+    mamba install -c conda-forge natcap.invest
 
-Details
--------
-Here is an explanation of what the commands are doing:
+If you prefer to use ``conda``, the command is otherwise the same::
 
-1. Create a brand-new environment with the correct python version.
+    conda install -c conda-forge natcap.invest
 
-   ``conda create -y -c conda-forge -n <name> python=<python version>``
+Installing with ``pip``
+-----------------------
 
-   To be safe, you should **always install** ``natcap.invest`` **into a brand-new virtual environment**. This way you can be sure you have all the right versions of dependencies. Many issues with installing or using the ``natcap.invest`` package arise from dependency problems, and it's a lot easier to create a new environment than it is to fix an existing one.
+The ``natcap.invest`` package is also available from the
+`Python Package Index <https://pypi.org/project/natcap.invest/>`_
+and installable with ``pip``.  Binary builds of ``natcap.invest`` are available for
+64-bit x86 architectures and also as a source distribution.  Note that if you
+have a computer with a non-x86-compatible architecture (such as a Mac with
+one of the M-series chips), you will either need to have a compiler installed
+or you will want to install ``natcap.invest`` from ``conda-forge`` (see above).
 
-2. Activate the brand-new environment just created.
+To install ``natcap.invest`` via ``pip``::
 
-   ``conda activate <name>``
+    pip install natcap.invest
 
-   If you run ``conda list`` after this, you'll see the specified python version is there along with around 15 other packages that are included with python by default. None of these are specific to invest. You're now in an isolated environment so you can control which versions of dependencies are available to invest.
+Note that ``natcap.invest`` depends on GDAL, which must already be compiled and
+available on your system.  GDAL can be installed from your system package manager,
+from ``conda``/``mamba``, or from
+`Christoph Gohlke's python builds for Windows <https://github.com/cgohlke/geospatial.whl/>`_.
```

Comment:
Is it still worth mentioning the Gohlke builds, given that GDAL binaries for Windows are available from conda-forge (at least semi-officially, it's recommended on the [GDAL installation page](https://gdal.org/download.html#binaries_))?